modular_skyrat/modules/modular_reagents/code/reagents/medicine.dm

```diff
@@ -25,6 +25,83 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time, 80)
 	..()
 
+
+/datum/reagent/medicine/ataraxydone
+	name = "Ataraxydone"
+	description = "A stasis-inducing chemical used for exotic surgeries or emergency medicine."
+	taste_description = "liquid"
+	reagent_state = LIQUID
+	color = "e0d394" // 224, 211, 148
```

Comment:
Don't colors work with a `#` in front of them? I forget.

Reply:
Worked without, but I put one in for consistency. Still seems good.